.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: CI Build
         run: ./scripts/cibuild
 
-      - uses: aws-actions/configure-aws-credentials@v1
+      - uses: aws-actions/configure-aws-credentials@v4
```

Comment:
This is not the culprit, but updating the version to the latest is nice and does not break anything.